### PR TITLE
docs: Major revisions to README.md, CODE_OF_CONDUCT.md, and CONTRIBUTING.md 

### LINKED ISSUE #66: docs: Clean up README, CODE_OF_CONDUCT, and CONTRIBUTING
All of the "meta" files in the repository need to be cleaned up to be more concise, readable, and project-specific; the last two are currently 90% templates. 

Will be doing this over the next few days.

### PR DESCRIPTION
**Pull request by mcao**
Resolves #66 

This pull request is a major revamp to our README and contribution guidelines, which should make it easier for people to contribute going forward! Yeet

### CODE REVIEW

**Review by neeltron — APPROVED** [review]
sike, couldn't have done it better

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,129 +1,50 @@
-# Contributing to this repository 
+# Contributing to Pridebot
 
-## Getting started 
+## Getting started
 
-Before you begin:
-- This bot is powered by Discord.py
-- Check out the [existing issues](https://github.com/tyffical/Pridebot/issues) & see if we accept contributions
+### **Before you begin:** Please contact one of the Pridebot maintainers in the BLAHAJGang Discord Server who will be able to walk you through this repository and how you can best contribute! Otherwise, you might risk working on something that's already in progress or going through unnecessary effort to get your contributions approved.
```
Comment (tyffical) [inline CONTRIBUTING.md:5]: Not sure how i feel about this whole section being bolded in terms of readability. You can add emphasis in other ways?

File: README.md [hunk 2]
```diff
@@ -26,39 +30,36 @@
   - [Resources](#resources)
 - [Maintainers](#maintainers)
 - [Other Major Contributors](#other-major-contributors)
```
Comment (tyffical) [inline README.md:32]: Oop should remove this line from the table of contents too, since we removed the section

**Review by tyffical — CHANGES_REQUESTED** [review]
Besides those minor comments, this looks awesome Michael! Good job!

**Review by tyffical — APPROVED** [review]

**Pull request merged**